cri_lib/cri_controller.py

```diff
@@ -13,6 +13,7 @@
 
 from .cri_errors import CRICommandTimeOutError, CRIConnectionError
 
+logger = logging.getLogger(__name__)  # Retrieves the logger from __init__.py
 
 class CRIController:
```

Comment:
The `__name__` here is going to be `"cri_controller"`. See [this explanation](https://www.pythontutorial.net/python-basics/python-__name__/).
```suggestion
logger = logging.getLogger(__name__)

class CRIController:
```

Reply:
Logger name will be "cri_lib.cri_controller" when imported